### ISSUE TITLE
Knowledge export C: define synthetic dataset and training-run governance

### ISSUE THREAD

**Issue by pokekarten** [pokekarten/igentic-iphone#82]
Parent: #79

## Goal

Document a reproducible, provider-independent path from synthetic data creation to LoRA/QLoRA experiment review while protecting the immutable benchmark from contamination.

## Allowed files

```text
docs/model-research/DATASET_GOVERNANCE.md
docs/model-research/TRAINING_RUN_CONTRACT.md
docs/model-research/training-run-manifest-v0.example.json
```

Change no other files.

## Dataset governance requirements

Define:

- test, validation and training split ownership;
- immutable benchmark isolation;
- prohibition on copying, paraphrasing or generating from test cases;
- synthetic-only initial training data;
- provenance, generator version, reviewer status and license fields;
- unique IDs, hashes, schema versions and change history;
- duplicate and near-duplicate detection expectations;
- German/English balance and category coverage;
- privacy, redaction and publication rules;
- rejection rules for unsafe, private, contaminated or ambiguous records.

## Training-run contract requirements

Every run must record:

- immutable model ID and revision;
- exact license reference and gate status;
- dataset IDs, revisions and hashes;
- immutable test-set ID;
- tokenizer/chat/tool template revisions;
- environment and dependency lock;
- seed, context, output limit, batch and accumulation;
- LoRA targets, rank, alpha, dropout and quantization settings;
- checkpoint and resume locations without credentials;
- untouched baseline metrics and post-training metrics;
- export targets and runtime compatibility status;
- explicit `KEEP`, `REWORK` or `REJECT` decision with reason.

The provider is an executor only. Kaggle, Colab or other free credits must be interchangeable and never become a required dependency.

## Acceptance criteria

- [ ] Exactly three allowed files are changed.
- [ ] Benchmark contamination is explicitly prevented.
- [ ] Public and private data boundaries are explicit.
- [ ] Run manifests are provider-independent and resumable.
- [ ] FunctionGemma 270M is the default first specialization candidate.
- [ ] Qwen3 0.6B training occurs only after untouched comparison evidence.
- [ ] No weights, adapters, checkpoints, credentials or private data are committed.
- [ ] No training result or free-compute availability is claimed.
- [ ] `python3 scripts/validate_repo_structure.py` succeeds.

## Stop rules

Do not begin training, download gated artifacts, alter benchmark V0, add notebooks with credentials, add model dependencies or claim iPhone Air readiness. Start only after the evaluator contract is terminal.

**Comment by pokekarten** [comment]
## Abschlussprüfung

Issue #82 ist auf `main` umgesetzt.

Evidence:
- PR #95 (`Docs: add dataset and training-run governance`) wurde gemergt.
- Der PR änderte exakt die drei erlaubten Dateien.
- Der exakte PR-Head hatte grüne Pflicht-Workflows für Scope, PR-Qualität, Repo-Audit, Phase-0-Validierung und Docs-Konsistenz.
- `DATASET_GOVERNANCE.md` trennt immutable Test-, Validation- und Training-Splits, verbietet Kopieren/Paraphrasieren/Generieren aus Benchmarkfällen und dokumentiert Provenienz, Hashes, Review, Sprach-/Kategorieabdeckung sowie Ablehnungsregeln.
- `TRAINING_RUN_CONTRACT.md` ist provider-unabhängig, nennt FunctionGemma 270M als ersten Spezialisierungskandidaten und erlaubt Qwen3 0.6B erst nach untouched baseline evidence.
- Das Beispielmanifest verwendet den kanonischen Modellnamen `google/functiongemma-270m-it`, enthält Owner und Erstellungszeit-Platzhalter, keine Credentials/Weights/Checkpoints und behauptet weder Trainingsergebnis noch Gerätebereitschaft.

Damit sind die Akzeptanzkriterien erfüllt. Der nächste geordnete Knowledge-Export-Schritt ist #83.
